src/test/java/org/folio/domain/MonetaryValueTest.java

```diff
@@ -0,0 +1,110 @@
+package org.folio.domain;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.math.BigDecimal;
+
+import static org.junit.Assert.*;
+
+@RunWith(JUnitParamsRunner.class)
+public class MonetaryValueTest {
+
+  @Test(expected = NullPointerException.class)
+  public void stringConstructorThrowsExceptionWhenAmountIsNull() {
+    new MonetaryValue((String) null);
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void doubleConstructorThrowsExceptionWhenAmountIsNull() {
+    new MonetaryValue((Double) null);
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void bigDecimalConstructorThrowsExceptionWhenAmountIsNull() {
+    new MonetaryValue((BigDecimal) null);
+  }
```

Comment:
What is the goal of these tests?

Reply:
These are unit tests for `MonetaryValue` class, I think they were copy/pasted from `mod-feesfines` along with the class itself.